### DISCUSSION TITLE
关于JUNO大作业里面argparse相关指令的询问

### DISCUSSION THREAD

**Discussion by CreatinGL** [physics-data/faq#154] [Q&A] [answered]
助教老师你好！我和同学这几天一直被调用argparse里面指令来完成绘图所困扰，网上的相关文章大多读起来比较晦涩。能麻烦详细地讲一下怎么使用这些指令吗？谢谢！

**Comment by Harry-Chen** [comment]
@Berrysoft

**Comment by Berrysoft** [comment, marked as answer]
我在给出的模板中已经有使用方法；也可以去查阅[详细文档](https://docs.python.org/3/library/argparse.html)。

最基础的情况下，你只需要考虑使用 parse 的结果 `args` 变量，其中的各个成员由 `parser.add_argument` 调用中的 `dest` 参数给出。
例如，对于参数
``` python
parser.add_argument("-g", "--geo", dest="geo", type=str, help="Geometry file")
```
短参数为 `-g`，长参数为 `--geo`，参数类型为字符串，传入的参数可以使用 `args.geo` 获取。

而且事实上对于 `draw.py`，只需要修改 `TODO` 所在的函数，其他地方都不需要更改，我都写好了；调用该文件的命令也已经在 `Makefile` 里面写好了。
